src/components/PatentUpload/PatentUpload.js

```diff
@@ -37,16 +37,17 @@ class PatentUpload extends Component {
             /* If the input passes, iterate through all the files the user uploads
                 to the S3 bucket in a folder named public and then the protein name the user input
                 ex. s3bucket/public/PCSK9/1234.pdf */
+            
             Promise.all(Array.from(input.files).map((file) => {
-                return Storage.put(`${proteinName.toLowerCase()}/${file.name}`,
```

Comment:
the return change shouldn't make a difference here. I think the issue was the file type

Reply:
yeah this was accidentally left behind from some of the other solutions that I had been trying